### PR TITLE
Update to gold standards

### PR DESCRIPTION
**Pull request by dushaniw**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: jira7/Package.md [hunk 1]
```diff
@@ -1,72 +1,94 @@
...
+|                    |    Version     |  
+| :-----------------:|:--------------:| 
+| Ballerina Language | 0.970.0-beta15 |
+| JIRA REST API      |    7.2.2       |  
 
-    ```ballerina
-       import wso2/jira7;
-    ```
+## Sample
+The JIRA connector can be used to get summaries of all projects, create a project, and create an issue. First, import
```
Comment (appellj) [inline jira7/Package.md]: Delete "The JIRA connector can be used to get summaries of all projects, create a project, and create an issue."

File: jira7/Package.md [hunk 2]
```diff
@@ -1,72 +1,94 @@
...
+the `wso2/jira7` package into the Ballerina project.
+```ballerina
+import wso2/jira7;
+```
+**Obtaining Credentials to Run the Sample**
 
-5. Provide the credentials to your endpoint in the initialization step, and use as shown 
-in the following sample code.
+1. Visit [Atlassian](https://id.atlassian.com/signup) and create an Atlassian Account.
+2. Obtain the following credentials from the Twilio dashboard:
```
Comment (appellj) [inline jira7/Package.md]: Step 2 mentions Twilio, not JIRA. Please review this section and make sure it's for JIRA.

File: jira7/Package.md [hunk 3]
```diff
@@ -1,72 +1,94 @@
...
-           var result = jiraEndpoint -> getProject(projectKey);
-           match result{
-               jira:Project p => project = p;
-               jira:JiraConnectorError e => jiraError = err;
-           }
-       }
-        
-    ```
\ No newline at end of file
+You can now enter the credentials and base URL in the HTTP client config.
```
Comment (appellj) [inline jira7/Package.md]: The base URL is specified outside of the HTTP client config. Is this text or the sample incorrect?

File: jira7/Package.md (hunk 1 above)
Comment (dushaniw) [reply jira7/Package.md]: Fixed.

File: jira7/Package.md (hunk 2 above)
Comment (dushaniw) [reply jira7/Package.md]: Checked and added the correct instruction.

File: jira7/Package.md (hunk 3 above)
Comment (dushaniw) [reply jira7/Package.md]: Modified the code to get the base URL inside the HTTP client config. Updated the sample.

**Pull request merged**